source.mak

```diff
@@ -46,6 +46,7 @@ PARSER_SOURCES =				\
 	$(PARSER_DIR)/cobol.c			\
 	$(PARSER_DIR)/diff.c			\
 	$(PARSER_DIR)/dosbatch.c		\
+	$(PARSER_DIR)/dts.c			\
```

Comment:
A similar change is required in `win32/ctags_vs2013.vcxproj` and `win32/ctags_vs2013.vcxproj.filters`, such that we at least try not to break win32 build.